docs/en-US/component/backtop.md

```
@@ -33,7 +33,7 @@ backtop/custom
 
 | Name              | Description                                                          | Type      | Default |
 | ----------------- | -------------------------------------------------------------------- | --------- | ------- |
-| target            | the target to trigger scroll.                                        | ^[string] | —       |
+| target  | the target to trigger scroll. Version ^(2.13.1) and later support `window` and `HTMLElement`.  | `string` \| `HTMLElement` \| `Window ` | —        |
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)